operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -563,10 +563,10 @@ spec:
       - description: Custom environment variables to set on managed pods' containers.
         displayName: Environment Variables
         path: customize.envVars
-      - description: Settings for the Node-Scanner container, which is responsible
-          for scanning the Nodes' filesystem
+      - description: Settings for the Node-Inventory container, which is responsible
+          for scanning the Nodes' filesystem.
         displayName: Node Scanner Settings
```

Comment:
👍 for keeping the `displayName: Node Scanner Settings`

Reply:
It is now. _Node Scanning Settings_ 

[0296560](https://github.com/stackrox/stackrox/pull/6160/commits/0296560e291e5a758622ace57dcc4d7ec9ab00ba)